Jenkinsfile-deploy

```diff
@@ -1,36 +1,8 @@
 library("tdr-jenkinslib")
 
-pipeline {
-    agent {
-        label "master"
-    }
-    parameters {
-        choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are building the api update lambda for")
-        string(name: "TO_DEPLOY", description: "The git tag, branch or commit reference to deploy, e.g. '1'")
-    }
-    stages {
-        stage("Deploy lambda") {
-            agent {
-                ecs {
-                    inheritFrom "aws"
-                    taskrole "arn:aws:iam::${env.MANAGEMENT_ACCOUNT}:role/TDRJenkinsNodeLambdaRole${params.STAGE.capitalize()}"
-                }
-            }
-            steps {
-                script {
-                    def accountNumber = tdr.getAccountNumberFromStage(params.STAGE)
-                    sh "python3 /deploy_lambda_from_s3.py ${accountNumber} ${params.STAGE} tdr-api-update-${params.STAGE} tdr-backend-code-mgmt ${params.TO_DEPLOY}/api-update.jar"
-                }
-            }
-        }
-    }
-    post {
-      success {
-        script {
-          if (params.STAGE == "intg"){
-            tdr.runEndToEndTests(0, params.STAGE, BUILD_URL)
-          }
-        }
-      }
-    }
-}
+deployToLambda(
+        stage: params.STAGE,
+        libraryName: "tdr-api-update",
```

Comment:
Is this the correct `libraryName `value to be passing in.

Looking at the latest code on the[ jenkinslib master branch](https://github.com/nationalarchives/tdr-jenkinslib/blob/master/vars/deployToLambda.groovy), the `deployToLamda` function uses this value as follows:

 `sh "python3 /deploy_lambda_from_s3.py ${accountNumber} ${config.stage} tdr-${config.libraryName}-${config.stage} `

So the deploy lambda python function will have this passed in which doesn't seem correct: `tdr-tdr-api-update`

Reply:
Good spot, thanks. That parameter needs to match the Lambda name, which in this case is `tdr-api-update-<stage>`, so I've changed the parameter to `api-update`.